### DISCUSSION TITLE
Add-PnPSiteScript returns (triggerFlow) is invalid

### DISCUSSION THREAD

**Discussion by myatix** [pnp/powershell#2577] [Q&A]
Hi All,

I have a little issue when I try to add a very simple Site Script... It would appear that the structure of LogicApp URL's have been changed and I am getting the following error???

Add-PnPSiteScript -Title $siteScriptTitle -Content $content
**Action 1 (triggerFlow) is invalid. Parameter url has an invalid value.**

This is the LogicApp URL that is created in Azure????
https://xxxx.azurewebsites.net:443/api/SiteProvisioning/triggers/manual/invoke?api-version=2022-05-01&sp=%2Ftriggers%2Fmanual%2Frun&sv=1.0&sig=vg56zW6APeXXXXXXXXX_KFb2lUo61X772a9UhVRg

Site Script:

```
{
    "$schema": "schema.json",
    "actions": [
        {
            "verb": "triggerFlow",
            "url": "https://xxxx.azurewebsites.net:443/api/SiteProvisioning/triggers/manual/invoke?api-version=2022-05-01&sp=%2Ftriggers%2Fmanual%2Frun&sv=1.0&sig=vg56zW6APeXXXXXXXXX_KFb2lUo61X772a9UhVRg", 
            "parameters": 
            { 
               "event": "Microsoft Event", 
               "product": "SharePoint" 
            }
        }
    ], 
    "bindata": { }, 
    "version": 1
}
```

Can someone tell me if this is a bug or am I doing something wrong?

**Comment by Robert1976** [comment]
I am having the same issue. Any updates on this?

**Reply by glapointe** [reply]
Has anyone been able to get this to work? I've created a simple Flow and tried with "Anyone" and "Any user in the tenant" and nothing seems to work. I can call the endpoint directly using Invoke-RestMethod and that works fine but attempting to register the site script always throws the error.
